commit 5d8c7776343b3f1b96ef7777e4504378f23c041a
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Tue Apr 12 22:14:34 2022 +0200

    hurd: Fix arbitrary error code
    
    ELIBBAD is Linux-specific.
    
    (cherry picked from commit 67ab66541dc1164540abda284645e38be90b5119)

diff --git a/nss/nss_test_errno.c b/nss/nss_test_errno.c
index 680f8a07b97fe263..59a5c717bebd296f 100644
--- a/nss/nss_test_errno.c
+++ b/nss/nss_test_errno.c
@@ -28,7 +28,7 @@ static void __attribute__ ((constructor))
 init (void)
 {
   /* An arbitrary error code which is otherwise not used.  */
-  errno = ELIBBAD;
+  errno = -1009;
 }
 
 /* Lookup functions for pwd follow that do not return any data.  */
